src/app/checkout/checkout.component.js

```diff
@@ -112,6 +112,7 @@ class CheckoutController {
         })
         .subscribe((data) => {
           this.orderService.addFeesToNewGiftIfNecessary(data)
+          this.orderService.storeCartData(data)
```

Comment:
So, `addFeesToNewGiftIfNecessary` actually calls `storeCartData` so I need to remove these that I added.

Reply:
Ah yep in the ```updatePrices``` function.